tests/unit/broccoli/default-packager/styles-test.js

```diff
@@ -70,7 +71,9 @@ describe('Default Packager: Styles', function() {
   }));
 
   afterEach(co.wrap(function *() {
-    yield output.dispose();
+    if (output) {
```

Comment:
typically that means that `output` wasn't assigned, and if tests work correctly, we don't need this `if`. why did you add it?

Reply:
Failing tests hide the original error when output is undefined, that's why ... mean a cleanup can be performed only on smth that exists. Not an important change though.